lib/ui/screen/home_drawer.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:genericwidgetapp/ui/widgets/custom_list_tile.dart';
+
+class HomeDrawer extends StatelessWidget {
+
+  Widget container(){
+    return Container(
+      width: double.infinity,
+      padding: EdgeInsets.all(20),
+      color: Colors.deepPurple,
+      child: Center(
+        child: Column(
+          children: <Widget>[
+            Container(
+              width: 100.0,
+              height: 100.0,
+              margin: EdgeInsets.only(top: 30.0, bottom: 10.0),
+              decoration: BoxDecoration(
+                shape: BoxShape.circle,
+                image: DecorationImage(image: AssetImage("assets/images/person1.png"),
+                fit: BoxFit.fill
+                ),
+              ),
+            ),
+            Text("Name", style: TextStyle(fontSize: 22.0, color: Colors.white),)
+          ],
+        ),
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
Why build is returning widget .. according to our coding standards build should not return widget ..